### DISCUSSION TITLE
Where can I find out which version of gitlens works with which version of VS Code ?

### DISCUSSION THREAD

**Discussion by nojetlag** [gitkraken/vscode-gitlens#3032] [Q&A]
I tried to install gitlens with my VS Code and get the brilliant error message "... is not compatible with VS Code 1.81.1" so I went to look on the gitlens change log and website but can't find information on what version of gitlens I will need for my version of VS Code. Where to find this information ?

**Comment by eamodio** [comment]
VS Code should already handle this for you. When you install GitLens from the extensions view you should be offered the latest GitLens version that works with your VS Code version.
